### ISSUE TITLE
`promiseTypeDelimiter: ""` is ignored and instead uses default delimiter

### ISSUE THREAD

**Issue by ramast** [pburtchaell/redux-promise-middleware#290]
Given a configuration like this

```es6
        createPromise({promiseTypeDelimiter: "", promiseTypeSuffixes: ['Loading', 'Success', 'Error']}),
```

One would except actions to look like `MyActionLoading` but instead it translates to `MyAction_Loading`

**Comment by pburtchaell** [comment]
Hey @ramast, good catch. I would also expect `MyActionLoading` in this case. Do you have bandwidth to put up a PR to fix this? My time is limited for the next couple weeks. 

In the meantime, does it work to use `promiseTypeDelimiter: null`?

**Comment by ramast** [comment]
does it work to use `promiseTypeDelimiter: null` ? No and it won't work with any falsey value (null, '', undefined, false, 0, ...).
I've submitted a pull request https://github.com/pburtchaell/redux-promise-middleware/pull/291

**Comment by pburtchaell** [comment]
Thanks, @ramast. I'll give the PR a review as soon as possible.

**Comment by pburtchaell** [comment]
Merged #291 and released to npm as `6.1.3`.
